.github/workflows/build.yml

```diff
@@ -84,6 +84,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       USE_NINJA: "1"
+      CC: "clang"
```

Comment:
Maybe add a comment that this is for the benefit of the wasm2c tests?

Reply:
Done.